feature/interface/singleton/otg_tests/singleton_test/README.md

```diff
@@ -41,6 +49,29 @@ a new testbed configuration with the desired port types.
   * Verify that the traffic flow from ATE port-1 to ATE port-2 is
             now working after the interface is back up.
 
+### RT-5.1.3 [TODO: https://github.com/openconfig/featureprofiles/issues/2320]
+####  Subscribe to interface counters with SAMPLE mode:
+
+*   Test with two different SAMPLE intervals of 10 and 15 seconds
```

Comment:
I think the intent is to have two separate tests, one with 10 second sample, and a separate subtest with 15 second sample.    This is because we want to see which sample rates the DUT supports.

As written it could be interpreted to be one test with two simultaneous subscribes.

Reply:
reworded